client/src/pages/Client/index.tsx

```diff
@@ -1,95 +1,220 @@
-  jobId: 6,
-  freelancerId: 1,
-  createdAt: '2022-10-25T07:51:50.064Z',
-  updatedAt: '2022-10-25T07:51:50.064Z',
-  job: {
-    title: 'Back End Developer',
-  },
-  freelancer: {
-    id: 1,
-    userId: 1,
-    user: {
-      name: 'Ahmed',
-    },
-  },
-};
-
-const proposals = [
-  proposal, proposal,
-];
-
-const job = {
-
-  // 9
-  id: 6,
-  title:
-    'I need someone clean up a voice recording from static noise and other background noises',
-  description:
-    'I have voice recordings I can’t hear clearly sone words are clear I need someone boost up the voices remove static background noice so voices can clearly be herd',
-  // category: 'Music & Audio',
-  budget: 900,
-  // time: '2 months',
-  isOccupied: false,
-  userId: 2,
-  proposals,
-
-};
+import {
+  JobSearch, TabListInt,
+} from '../../interfaces';
+import './style.css';
 
 function Client() {
-  const { id } = useParams();
   const [showModel, setShowModel] = useState(false);
   const handleOpen = () => setShowModel(true);
   const handleClose = () => setShowModel(false);
+  const [message, setMessage] = useState('');
+  const [accept, setAccept] = useState('');
+  const [openAlert, setOpenAlert] = useState(false);
+  const [openAlertProposal, setOpenAlertproposal] = useState(false);
+  const [loading, setLoading] = useState(false);
+  const [jobsUnoccupied, setJobsUnoccupied] = useState<JobSearch[]>([]);
+  const [jobsOccupied, setJobsOccupied] = useState<JobSearch[]>([]);
+
+  useEffect(() => {
+    const getData = async () => {
+      setLoading(true);
+      try {
+        const { data } = await axios.get('/api/v1/client');
+        setLoading(false);
+        const { unOccupiedJobs } = data.data;
+        const { occupiedJobs } = data.data;
+        setJobsUnoccupied(unOccupiedJobs); // job isOccupied = fasle
+        setJobsOccupied(occupiedJobs); // // job isOccupied = true
+      } catch (err) {
+        setLoading(false);
+      }
+    };
+
+    getData();
+  }, []);
+
+  // fun delete item
+  const handleDelete = async (idItem: number) => {
+    try {
+      const dataItemDel = await axios.delete(`/api/v1/jobs/${idItem}`);
+      setMessage(dataItemDel.data.msg);
+      const filterJobsDelete = jobsUnoccupied.filter((job) => job.id !== idItem);
+      setJobsUnoccupied(filterJobsDelete);
+    } catch (err) {
+      setLoading(false);
+    }
+    setOpenAlert(true);
+  };
+
+  // fun Accipt proposal
+  const acceptPropsal = async (proposalID: number, jobID: number) => {
+    try {
+      const getProposalItem = await axios.patch(`/api/v1/proposals/${proposalID}`);
+
+      setAccept(getProposalItem.data.msg);
+      const jobAccepted = jobsUnoccupied.find((job) => job.id === jobID);
+      if (jobAccepted) {
+        const filterProposal = jobsUnoccupied.filter((job) => job.id !== jobID);
+        setJobsUnoccupied(filterProposal);
+        setJobsOccupied([...jobsOccupied, jobAccepted]);
+      }
+    } catch (err) {
+      setLoading(false);
```

Comment:
When you set loading true so, you set it `false` here,
Many errors can happen, I suppose that there's a better way of handling error!